sandbox-backend/src/main/resources/db/migration/V0000026__addEmails_Subscriptions.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE employee_category(
+    id          INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    categoryId  INT NOT NULL,
+    employeeId  INT NOT NULL,
+    FOREIGN KEY (categoryId)        REFERENCES category (id),
+    FOREIGN KEY (employeeId)        REFERENCES employee (id),
+    CONSTRAINT FK_employee_category UNIQUE (categoryId, employeeId)
+);
+
+ALTER TABLE employee ADD COLUMN email VARCHAR(320);
+
+UPDATE employee SET email='exadelovichjhon@gmail.com' WHERE id=1;
```

Comment:
OPTIONAL: I understand that in our application we can use ids but we don't have guarantee that we will have such ids on each server. So it it better use other identifiers in such cases, for example unique username of user.

Reply:
yea, agree